### ISSUE TITLE
Autoprefixer not running for Tailwind

### ISSUE THREAD

**Issue by delanebob** [remix-run/remix#7276]
### What version of Remix are you using?

1.19.3

### Are all your remix dependencies & dev-dependencies using the same version?

- [X] Yes

### Steps to Reproduce

I started a brand new Remix project yesterday with deploy to Cloudflare pages selected (don't believe deployment is relavent in this scenario).

I then followed the steps to install both [Tailwind](https://remix.run/docs/en/1.19.3/guides/styling#tailwind-css) & [Autoprefixer](https://remix.run/docs/en/1.19.3/guides/styling#postcss) as [this issue](https://github.com/remix-run/remix/issues/5475) mentions that it doesn't run automatically for Tailwind.

### Expected Behavior

When looking at the output CSS for Tailwind I would expect it to include webkit prefixes like `-webkit-backdrop-filter`.

### Actual Behavior

Looking at the Tailwind file, you can see the output is like below and doesn't include `-webkit-backdrop-filter`

```css
.backdrop-filter{
  backdrop-filter:var(--tw-backdrop-blur) var(--tw-backdrop-brightness) var(--tw-backdrop-contrast) var(--tw-backdrop-grayscale) var(--tw-backdrop-hue-rotate) var(--tw-backdrop-invert) var(--tw-backdrop-opacity) var(--tw-backdrop-saturate) var(--tw-backdrop-sepia)
}
```

**Comment by msaaron** [comment]
I've been running into the same issue after selecting Express as the deployment target. I spun up a new project, selected Remix App Server, and it worked fine

**Comment by markdalgleish** [comment]
It looks like PostCSS config errors are being swallowed by Remix, otherwise you'd see that there's an issue loading the config. I've opened a PR to ensure these errors are surfaced: https://github.com/remix-run/remix/pull/7391

To fix this, rename `postcss.config.js` to `postcss.config.cjs`. The issue is due to the use of `module.exports` in the PostCSS config when the project is configured to use ES Modules. This explains why this issue depends on the template being used. The docs for this have already been updated in v2 which will be going out soon.
